### ISSUE TITLE
NVDA doesn't read table inside modal

### ISSUE THREAD

**Issue by afiorenza** [nvaccess/nvda#5826]
Hi, I have a some text and a table inside a dialog, when dialog opens i wan't all content to be read but nvda only reads the text. Do i have to add some role or something to the table?

Thanks.

**Comment by jcsteh** [comment]
The question is: do you just want the text of the table to be read or do 
you want it to actually be read like a table, including headers, 
coordinates, etc.? If the latter, that suggests you want your dialog 
content to be treated as a document (i.e. navigable with browse mode, 
read like a document, etc.). To do that, put an element with 
role="document" inside the dialog and set focus to that when the dialog 
opens.

If you really don't want to treat it like a document, then only text 
would be read.

**Comment by afiorenza** [comment]
Thanks!
